backstop_history/BackstopHistory.py

```diff
@@ -475,25 +484,36 @@ def Assemble_History(self, ofls_dir, tbegin, interrupt=False):
 
         # WHILE
         # The big while loop that backchains through previous loads and concatenates the
-        # proper load sections and possibly events (e.,g. maneuivers)to the Master List
+        # proper load sections and possibly events (e.,g. maneuvers)to the Master List
         while self.master_ToFC > tbegin_time:
-
+            
             # Obtain the Continuity information of the present ofls directory
             cont_load_path, present_load_type, scs107_date = self.get_continuity_file_info(present_ofls_dir)
 
             self.logger.debug('    Cont load path: %s, load type: %s, scs107 date: %s' % (cont_load_path, present_load_type, scs107_date))
 
 
             # Read the commands from the Continuity file
-            cont_cr_cmds, cont_file_name = self.get_CR_bs_cmds(cont_load_path)
+            # The value of present_load_type indicates whether to read the CR*.backstop file
+            # or the VR*.backstop file.
+            
+            # Indicators of load types which require reading the VR*.backstop file
+            self.vehicle_types = ['VO_NORMAL', 'VO_TOO', 'VO_SCS-107', 'VO_STOP']
+            
+            if present_load_type.upper() in self.vehicle_types:
+                cont_cr_cmds, cont_file_name = self.get_VR_bs_cmds(cont_load_path)
+            else:
+                cont_cr_cmds, cont_file_name = self.get_CR_bs_cmds(cont_load_path)
+
 
             #---------------------- NORMAL ----------------------------------------
             # If the PRESENT (i.e. NOT Continuity) load type is "normal" then grab 
             # the continuity command set and concatenate those commands to the start of 
             # the Master List.
-            if present_load_type.upper() == 'NORMAL':
+            if (present_load_type.upper() == 'NORMAL') or\
```

Comment:
Shorter version of this:
```suggestion
            if present_load_type.upper() in ['NORMAL', 'VO_NORMAL']:
```